.github/workflows/build.yml

```diff
@@ -0,0 +1,120 @@
+name: Build
+
+on:
+  push:
+  schedule:
+    - cron: '0 4 * * *'
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+  linux:
+    strategy:
+      matrix:
+        arch: ['ia32', 'x64']
+    runs-on: ubuntu-latest
```

Comment:
We're skipping alpine for now since the images aren't yet available, right?

Reply:
Alpine is still running in a separate job, but it's failing for now until the images are released.